pkg/controller/manager/types_old.go

```diff
@@ -177,3 +186,60 @@ func unescapeFilters(strs []string) []string {
 	}
 	return strs
 }
+
+// parseStartDate parses the supplied string as a strfmt.DateTime.
+func parseStartDate(value string) (strfmt.DateTime, error) {
```

Comment:
ditto, why manager logic interpreting start date is part of user package?

Reply:
it's not related to manager really - it used to be that way, but it was moved to sctool. https://github.com/scylladb/scylla-manager/blob/d96e5d8338dd0afa5e130ee8cbb88b9cb11bc94b/pkg/command/flag/type.go#L102. This is essentially parsing a value from ScyllaCluster API, which has the same syntax

Reply:
I see, I though it's part of Scylla Manager API, but looks like these date and size extensions are part of CLI logic.